src/directives/ctr-input.ts

```diff
@@ -157,6 +158,19 @@ export class CtrInput {
         }
     }
 
+    @HostListener("click", ["$event"])
+    public onClick(event: any) {
+        this.el.nativeElement.select();
```

Comment:
At the moment select will happen even if `openOnClick` is false.
I thought we should have 2 options: `openOnClick` and `selectOnClick` so that user will have full control on click behaviour.